eman2/__init__.py

```diff
@@ -142,3 +160,10 @@ def defineBinaries(cls, env):
                        tar='eman2.31.linux64.tgz',
                        commands=eman231_commands,
                        default=True)
+
+        env.addPackage('eman', version='3.0.0-alpha',
```

Comment:
Why not using the constant here?

Reply:
And I think the hyphen between 0 and alpha could be problemeatic....I'd remove it: 3.0.0.alpha?